.travis.yml

```diff
@@ -1,30 +1,17 @@
 language: ruby
-os: osx
-env: OSX=10.11
-osx_image: xcode7.3
-rvm: system
+rvm: 2.0.0
+os: linux
 
-before_install:
-  - export TRAVIS_COMMIT=$(git rev-parse --verify -q HEAD)
-  - cd /usr/local
-  - sudo rm -rf /usr/local/.git/refs /usr/local/.git/packed-refs /usr/local/Library
-  - sudo rsync -az "$TRAVIS_BUILD_DIR/" /usr/local/
-  - sudo chown -R $USER /usr/local
-  - export TRAVIS_BUILD_DIR="/usr/local"
-  - if [ -f ".git/shallow" ]; then
-      travis_retry git fetch --unshallow;
-    fi
-  - git reset --hard $TRAVIS_COMMIT
-  - git clean -qxdff
-  - export HOMEBREW_DEVELOPER="1"
-  - ulimit -n 1024
```

Comment:
Are you sure none of this stuff from `before_install:` is needed for the regular Travis runs on macOS after a push to `master`?